include/openenclave/internal/trace.h

```diff
@@ -28,6 +28,17 @@ extern oe_log_level_t _log_level;
 #define OE_MAX_FILENAME_LEN 256U
 
 #if !defined(OE_BUILD_ENCLAVE)
+typedef void (*oe_log_callback)(
```

Comment:
Naming convention to disambiguate:
- `oe_log_callback_t` since this is a function typedef
- Non-static functions and variables shouldn't use the `_` prefix, so the variables become `oe_log_callback` and `oe_log_context`

Reply:
Fixed, thanks!